src/Lean/Elab/DeclModifiers.lean

```diff
@@ -26,6 +26,7 @@ def checkNotAlreadyDeclared {m} [Monad m] [MonadEnv m] [MonadError m] (declName
     if env.contains declName then
       throwError "a non-private declaration '{declName}' has already been declared"
 
+/-- Whether a declaration is regular, protected or private. -/
```

Comment:
Does this kind of doc comment really add value to the declaration? How about "Declaration visibility modifier" in this case?

Reply:
I think it does because you get this information when you hover it without needing to go-to-def.